garnett/gsdhoomdfilereader.py

```diff
@@ -289,7 +289,7 @@ def read(self, stream, frame=None):
         :type frame: :class:`trajectory.Frame`"""
         if NATIVE:
             try:
-                gsdfile = GSDFile(stream.name, stream.mode)
+                gsdfile = gsd.fl.open(stream.name, mode='rb')
```

Comment:
Why `'rb'` and not `stream.mode` as before?

Reply:
Fixed, thanks

Reply:
Actually, seems like I have to enforce the `'rb'` mode since it's the only way to open a file without having to specify application, schema and schema version.

Alternatively, we could just get the trajectory object directly with `gsd.hoomd.open` as @joaander suggested on #140. However, we would still need to have a gsdfile object since the `GSDHoomdFrame` takes one.